src/ApplicationInsights/ApplicationInsights/ApplicationInsights/NewApplicationInsightsLinkedStorageAccount.cs

```diff
@@ -52,13 +54,15 @@ public class NewApplicationInsightsLinkedStorageAccount : ApplicationInsightsBas
             Mandatory = true,
             ValueFromPipeline = true,
             HelpMessage = "PSApplicationInsightsComponent")]
+        [CmdletParameterBreakingChange("InputObject", ChangeDescription = "Parameter InputObject will be removed in upcoming Az.ApplicationInsights 2.0.0")]
```

Comment:
I remember InputObject is supported in generated cmdlet, could you confirm? @VeryEarly

Reply:
this input object is the parent resource which is not supported